src/panels/lovelace/editor/config-elements/glance-card/hui-glance-entity-editor.ts

```diff
@@ -0,0 +1,155 @@
+import { html, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+
+import { HuiEntityEditor } from "../../../components/hui-entity-editor";
+import { HomeAssistant } from "../../../../../types";
+import { ConfigEntity } from "../../../cards/hui-glance-card";
+import { GlanceOptions } from "./glance-options";
+import { EditorTarget } from "../../types";
+
+export class HuiGlanceEntityEditor extends HuiEntityEditor {
+  protected hass?: HomeAssistant;
+  protected entities?: ConfigEntity[];
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      entities: {},
+    };
+  }
+
+  protected renderOptions(
+    entityConf: ConfigEntity,
+    index: number
+  ): TemplateResult {
+    const glanceOptions = new GlanceOptions(entityConf);
+
+    return html`
+      <paper-toggle-button>Options</paper-toggle-button>
+      <div class="options">
+        <paper-input
+          label="Name"
+          .value="${glanceOptions._name}"
+          .configValue="${"name"}"
+          .index="${index}"
+          @value-changed="${this._optionChanged}"
+        ></paper-input>
+        <paper-input
+          label="Icon"
+          .value="${glanceOptions._icon}"
+          .configValue="${"icon"}"
+          .index="${index}"
+          @value-changed="${this._optionChanged}"
+        ></paper-input>
+        <paper-dropdown-menu label="Toggle Action">
+          <paper-listbox
+            slot="dropdown-content"
+            .selected="${glanceOptions._tapaction}"
+            attr-for-selected="action"
+            .configValue="${"tap_action"}"
+            .index="${index}"
+            @selected-item-changed="${this._optionChanged}"
+          >
+            <paper-item action="more-info">More Info Dialog</paper-item>
+            <paper-item action="toggle">Toggle</paper-item>
+            <paper-item action="call-service">Call Service</paper-item>
+          </paper-listbox>
+        </paper-dropdown-menu>
+        <paper-dropdown-menu label="Hold Action">
+          <paper-listbox
+            slot="dropdown-content"
+            .selected="${glanceOptions._holdaction}"
+            attr-for-selected="action"
+            .configValue="${"hold_action"}"
+            .index="${index}"
+            @selected-item-changed="${this._optionChanged}"
+          >
+            <paper-item action="">No Action</paper-item>
+            <paper-item action="more-info">More Info Dialog</paper-item>
+            <paper-item action="toggle">Toggle</paper-item>
+            <paper-item action="call-service">Call Service</paper-item>
+          </paper-listbox>
+        </paper-dropdown-menu>
+        <paper-input
```

Comment:
We should only show this when one of the actions is service?
```suggestion
        ${
          glanceOptions._holdaction === "call-service" ||
          glanceOptions._tapaction === "call-service"
            ? html`
                <paper-input
                  label="Service"
                  .value="${glanceOptions._service}"
                  .configValue="${"service"}"
                  .index="${index}"
                  @value-changed="${this._optionChanged}"
                ></paper-input>
              `
            : ""
        }
```